suzieq/poller/controller/source/netbox.py

```diff
@@ -115,10 +115,16 @@ def _init_session(self, headers: dict):
             headers ([dict]): headers to initialize the session
         """
         if not self._session:
-            self._session = aiohttp.ClientSession(
-                headers=headers,
-                connector=aiohttp.TCPConnector(verify_ssl=self._ssl_verify)
-            )
+            if self._ssl_verify:
+                self._session = aiohttp.ClientSession(
+                    headers=headers,
+                    connector=aiohttp.TCPConnector(ssl=None)
+                )
+            else:
+                self._session = aiohttp.ClientSession(
+                    headers=headers,
+                    connector=aiohttp.TCPConnector(ssl=False)
+                )
```

Comment:
```suggestion
            ssl_option = None if self._ssl_verify else False
            self._session = aiohttp.ClientSession(
                headers=headers,
                connector=aiohttp.TCPConnector(ssl=ssl_option)
            )
```
Agree with Dinesh, I think is better having something like this